src/apps/config/CMakeLists.txt

```diff
@@ -1,20 +1,4 @@
-project(config)
+# set up this later
+# or even better: completely remove
```

Comment:
Do you have plans to move display preferences to in-game settings?

Reply:
I don't think their are plans for that. The original config tools still work though.

Reply:
This extremely old config tool only operates on ini file. It does not depend on anything from the engine sources. At the same time, it requires MFC that no one wants to install. Finally, it does not really require any changes. It can be deployed just like we do it to libs.
What about our plans, I would like to have in-game preferences, yes. But as Hammie noticed we won't have it in the nearest future :)

Reply:
Original config tool mostly useless, because modern resolutions are not supported and you still need to set it up manually in .ini file, where you can configure other options as well. Furthermore, sound volume can be configured in game.

I don't think that this is important for nearest future, but most users (especially those not related to IT) prefer some kind of GUI instead of manual editing config files :)

![image](https://user-images.githubusercontent.com/11532502/125892936-7e94e941-2efa-447e-9826-15c87d88b620.png)

Reply:
I believe the one from TEHO is slightly modified to support new resolutions. But it is still just a bad way to manage in-game settings.
It is indeed on my to-do list, but we have just a few people working on the engine right now so I would not expect to see it until more important things are done...

Reply:
No problem, just wanted to clarify how it will work in the future (no matter how far away)